ert_shared/ensemble_evaluator/evaluator.py

```diff
@@ -136,6 +136,11 @@ async def evaluator_server(self, done):
         ):
             await done
             logger.debug("Got done signal.")
+            count = 0
+            while not self._snapshot.is_finished() and count < 10:
+                # give NFS adaptors some time to read last events
+                asyncio.sleep(1)
```

Comment:
I think https://github.com/equinor/ert/issues/1126 would make this trivial. If the EE owns the NFSAdaptor tasks, it will know when they have read all their events.

I'm fine with this, but I would add a big `TODO` or `XXX: this sleep is temporary, please forgive us`.

Reply:
Thanks for your comments. I changed the approach to instead wait for all who connect to /dispatch to disconnect. This gets rid of the sleep, and is a more general way to let any event that might be inbound get a chance to be processed.